test/src/utils/get_utils_test.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:get/get.dart';
+
+// Identity util to convert to iterables
+dynamic _id(dynamic e) => e;
+
+void main() {
+  // Tests for GetUtils.isNullOrBlank are located at dynamic extensions
+  group('GetUtils.isLength* functions', () {
+    test('isLengthEqualTo should validate iterable lengths', () {
+      // iterables should cover list and set
+      expect(GetUtils.isLengthEqualTo([].map(_id), 0), true);
```

Comment:
hey, just for me to understand, why have you used map here?
using simply a `[]` will not work?

Reply:
I wanted to test Iterables, to cover lists and sets.
I couldn't find a better way to make a iterable than using map